transcendence-app/src/chat/chat.service.ts

```diff
@@ -106,4 +107,34 @@ export class ChatService {
       },
     );
   }
+
+  async updateChatroom(
```

Comment:
praise: Looks very good :+1:

suggestion: We could check if the user is the chatroom owner before updating it.

Reply:
Like a double check, access to the edit details page only if admin or owner, and endpoint available only if admin or owner?

Reply:
I'll create an issue about the web admin role